iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/CatalogConstants.java

```diff
@@ -15,6 +15,8 @@ public final class CatalogConstants {
   /** Used to uniquely identify an update towards a table from user side. */
   public static final String COMMIT_KEY = "commitKey";
 
+  static final String STOP_TABLE_CREATION_TOGGLE = "stop_create";
```

Comment:
a future nice to have would be features that are type safe, perhaps all of the configuration for such features is stored in java in a schema that is then deployed to the database like in an ORM.

Then the column value of the feature can be changed at runtime.

Reply:
I tried adding a feature registry in the new PR: , See `cluster/configs/src/main/java/com/linkedin/openhouse/cluster/configs/TblPropsToggleRegistry.java`

Reply:
not exactly the thing that can be strongly typed in database schema. wanted to think about how to create constraints in mysql so that we could avoid leaving these feature Id being free-form string as much as possible.